lib/inc/pxp-agent/configuration.hpp

```diff
@@ -67,6 +69,8 @@ struct Entry : EntryBase
             : EntryBase { std::move(_name), std::move(_aliases), std::move(_help), std::move(_type) },
               value { std::move(_value) } {
     }
+
+    ~Entry() override = default;
```

Comment:
According to [my testing](https://gist.github.com/mruzicka/c527df43e92a578bc56e8bee7788ae58) this does not seem to be necessary as that is the what the compiler does by default. I.e. it generates a default destructor and since the destructor is declared virtual in the base class it must override that from the base class.

Reply:
Ah cool, I'll remove it.

Reply:
Removed.